service_capacity_modeling/models/org/netflix/evcache.py

```diff
@@ -261,9 +262,11 @@ def reserve_memory(instance_mem_gib):
         )
 
     ec2_cost = zones_per_region * cluster.annual_cost
+    spread_cost = 100000.0/float(cluster.count)
```

Comment:
I'd make it proportional to the existing cost rather than a fixed cost. So e.g. take the ec2 cost and add a decreasing tax by node count. Example
```
# g = number of nodes
tax = ec2_cost * g(x)
def tax(x, max_tax=0.10, min_tax=0.0):
    if x < 2:
        return max_tax
    elif x >= 32:
        return min_tax
    else:
        return min_tax + (max_tax - x * (max_tax - min_tax) / 30)
```